public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx

```diff
@@ -673,9 +676,19 @@ export const QueryAndExpressionsStep = ({ editingExistingRule, onDataChange }: P
 
       <ConfirmModal
         isOpen={showResetModeModal}
-        title="Switching to simple mode"
-        body="The selected queries and expressions cannot be converted to simple mode. Switching will remove them. Do you want to proceed?"
-        confirmText="Yes"
+        title="Deactivate advanced options"
+        body={
+          <div>
+            <Text element="p">
+              <Trans i18nKey="alerting.queryAndExpressionsStep.disableAdvancedOptions.text">
+                The selected queries and expressions cannot be converted. If you deactivate advanced options, your query
```

Comment:
It feels like "cannot be converted" is more vague. Is there a way we can phrase it that keeps that in still?

Reply:
you're right, let's put c`onverted to default`